src/NexusMods.DataModel/Synchronizer/SynchronizerService.cs

```diff
@@ -170,49 +170,51 @@ private IObservable<LoadoutSynchronizerState> CreateStatusObservable(LoadoutId l
 
         var revisions = Loadout.RevisionsWithChildUpdates(_conn, loadoutId)
             // Use DB transaction, since child updates are not part of the loadout
-            .Select(rev => (loadout:rev, revDbTx:_conn.Db.BasisTxId));
+            .Select(rev => (loadout: rev, revDbTx: _conn.Db.BasisTxId));
 
-        var statusObservable = Observable.CombineLatest(isBusy,
-                lastApplied,
+        var statusObservable = isBusy.CombineLatest(lastApplied,
                 revisions,
                 (busy, last, rev) => (busy, last, rev.loadout, rev.revDbTx)
             )
             .DistinctUntilChanged()
-            .SelectMany(
-                async tuple =>
+            .ToObservable()
```

Comment:
Should try and keep as much of the observable stream in R3. I'd suggest moving this further up.

Reply:
I changed most of the code in the function and called to be R3. I didn't change the interface signature as that would have required changing all the users and I didn't want to go too out of scope